doc/sphinx/source/recipes/recipe_kcs.rst

```diff
@@ -0,0 +1,141 @@
+
+Preprocessors: We've tried to use built-in preprocessor functions as much as possible. The first diagnostic requires global mean temperature anomalies for each dataset, both CMIP and the target model, and some multimodel statistics. The second diagnostic requires data on a point in the Netherlands. However, the ``extract_point`` preprocessor can be changed to ``extract_shape`` or ``extract_region``, in conjunction with an area mean. And of course, the coordinates can be changed to analyze a different region.
+
+Diagnostics:
+
+* global_matching
+
+  * ``scenario_years``: a list of time horizons. Default: ``[2050, 2085]``
+  * ``scenario_percentiles``: a list of percentiles for the steering table. Default: ``[p10, p90]``
+
+* local_resampling
+
+  * ``control_period``: the control period shared between all scenarios. Default: ``[1981, 2010]``
+  * ``n_samples``: the final number of recombinations to be selected. Default: ``8``
+  * ``scenarios``: a scenario name and list of options. The default setting is a single scenario:
+
+    .. code-block:: yaml
+
+        scenarios:
+          ML_MOC:  # scenario name; can be chosen by the user
+            description: "Moderate / low changes in seasonal temperature & precipitation"
+            global_dT: 1.0
+            scenario_year: 2050
+            resampling_period: [2021, 2050]
+            dpr_winter: 4
+            pr_summer_control: [25, 55]
+            pr_summer_future: [45, 75]
+            tas_winter_control: [50, 80]
+            tas_winter_future: [20, 50]
+            tas_summer_control: [0, 100]
+            tas_summer_future: [0, 50]
+
+    These values are taken from table 1 in the Lenderink 2014's supplementary material. Multiple scenarios can be processed at once by appending more configurations below the default one. For new applications, ``global_dT``, ``resampling_period`` and ``dpr_winter`` are informed by the output of the first diagnostic. The percentile bounds are to be tuned until a satisfactory selection is achieved.
+
+Example output
+--------------
+
+The diagnostic ``global_matching`` produces a scenarios table like the one below
+
+.. code-block:: python
+
+       year percentile  cmip_dt period_bounds  target_dt  pattern_scaling_factor
+    0  2050        P10     0.98  [2019, 2048]       0.99                    1.00
+    1  2050        P90     2.01  [2045, 2074]       2.02                    0.99
+    2  2085        P10     1.38  [2030, 2059]       1.38                    1.00
+    3  2085        P90     3.89  [2071, 2100]       3.28                    1.18
+
+
+which is printed to the log file and also saved as a csv-file ``scenarios.csv``.
+Additionally, a figure is created showing the CMIP spread in global temperature change,
+AND highlighting the selected steering parameters and resampling periods:
+
+.. _fig_kcs_global_matching:
+.. figure::  /recipes/figures/kcs/global_matching.png
```

Comment:
Important that this figure is correct, in my version (pdf) two changes are needed still (maybe these already happened?):
- target model, only 1 RCP scenario can be used at the same time, otherwise period selection does not make sense.
- resampling periods do not yet make sense (beyond 2100).

Reply:
That has been fixed. I can help you make a local build to preview the documentation.